farm/modeling/language_model.py

```diff
@@ -1614,3 +1619,99 @@ def enable_hidden_states_output(self):
 
     def disable_hidden_states_output(self):
         self.model.ctx_encoder.config.output_hidden_states = False
+
+
+class BigBird(LanguageModel):
+    """
+    A BERT model that wraps HuggingFace's implementation
+    (https://github.com/huggingface/transformers) to fit the LanguageModel class.
+    Paper: https://arxiv.org/abs/1810.04805
+
+    """
+
+    def __init__(self):
+        super(BigBird, self).__init__()
+        self.model = None
+        self.name = "big_bird"
+
+    @classmethod
+    def from_scratch(cls, vocab_size, name="big_bird", language="en"):
+        big_bird = cls()
+        big_bird.name = name
+        big_bird.language = language
+        config = BigBirdConfig(vocab_size=vocab_size)
+        big_bird.model = BigBirdModel(config)
+        return big_bird
+
+    @classmethod
+    def load(cls, pretrained_model_name_or_path, language=None, **kwargs):
+        """
+        Load a pretrained model by supplying
+
+        * the name of a remote model on s3 ("bert-base-cased" ...)
+        * OR a local path of a model trained via transformers ("some_dir/huggingface_model")
+        * OR a local path of a model trained via FARM ("some_dir/farm_model")
+
+        :param pretrained_model_name_or_path: The path of the saved pretrained model or its name.
+        :type pretrained_model_name_or_path: str
+
+        """
+
+        big_bird = cls()
+        if "farm_lm_name" in kwargs:
+            big_bird.name = kwargs["farm_lm_name"]
+        else:
+            big_bird.name = pretrained_model_name_or_path
+        # We need to differentiate between loading model using FARM format and Pytorch-Transformers format
+        farm_lm_config = Path(pretrained_model_name_or_path) / "language_model_config.json"
+        if os.path.exists(farm_lm_config):
+            # FARM style
+            big_bird_config = BigBirdConfig.from_pretrained(farm_lm_config)
+            farm_lm_model = Path(pretrained_model_name_or_path) / "language_model.bin"
+            big_bird.model = BigBirdModel.from_pretrained(farm_lm_model, config=bert_config, **kwargs)
```

Comment:
`bert_config` should be `big_bird_config`here. Otherwise `bert_config`is undefinded.

Reply:
Thanks!